contrib/kafka/pkg/adapter/adapter.go

```diff
@@ -0,0 +1,170 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kafka
+
+import (
+	"encoding/json"
+	"github.com/Shopify/sarama"
+	"github.com/bsm/sarama-cluster"
```

Comment:
this lib was yesterday marked as deprecated: https://github.com/bsm/sarama-cluster/commit/0762873379bf8be2b0193726813b88fa445ce95f

I guess we should move the sources, and the CCP over to use the native/shopify library

Reply:
I've sent a PR for this, here:

https://github.com/dubee/eventing-sources/pull/1